android/src/main/java/com/segment/analytics/kotlin/android/plugins/AndroidLifecyclePlugin.kt

```diff
@@ -63,6 +64,7 @@ class AndroidLifecyclePlugin() : Application.ActivityLifecycleCallbacks, Default
         packageInfo = try {
             packageManager.getPackageInfo(application.packageName, 0)
         } catch (e: PackageManager.NameNotFoundException) {
+            analytics.reportInternalError(e)
```

Comment:
Should we report the `AssertionError` instead of the incoming `NameNotFoundException`?